js/scripts/Car.js

```diff
@@ -0,0 +1,31 @@
+/**  @type {HTMLCanvasElement} */ 
+
+class Car{
+    constructor(x, y, w, h, ctx){
+    this.x= x
+    this.y= y   
+    this.w= w
+    this.h= h
+    this.ctx= ctx
+    this.speedX = 0
+    this.image = new Image
```

Comment:
Same logic I sent you before. To draw the car, you just have to create the image and then call the draw function on your main update!